wavelength-plugins/splatoon.js

```diff
@@ -0,0 +1,248 @@
+				}
+				if (Rooms("splatoon")) {
+					Rooms("splatoon").addRaw(`${WL.nameColor(user.name, true)} has disabled Splatfest.`);
+				}
+				// Clear splatfest teams
+				SPLATFEST.alpha = null;
+				SPLATFEST.bravo = null;
+			},
+
+			j: "join",
+			setteam: "join",
+			jointeam: "join",
+			join: function (target, room, user) {
+				if (!target) return this.parse(`/splatoonhelp`);
+				if (!SPLATFEST.active) return this.errorReply(`There is currently not a Splatfest. :(`);
+				if (!SPLATFEST.alpha === toId(target) || !SPLATFEST.bravo === toId(target)) return this.errorReply(`This is not a Splatfest team.`);
+				let splatProfile = Db.splatoon.get(user.userid);
+				splatProfile.splatfest = target;
+				Db.splatoon.set(user.userid, splatProfile);
+				return this.sendReply(`You have successfully joined Splatfest Team "${target}".`);
+			},
+
+			teams: "team",
+			team: function (target, room, user) {
+				if (!this.runBroadcast()) return;
+				if (!SPLATFEST.active) return this.errorReply(`There is currently not a Splatfest. :(`);
+				return this.sendReplyBox(`<strong>Splatfest Teams:</strong> ${SPLATFEST.alpha} and ${SPLATFEST.bravo}`);
+			},
+		},
+
+		name: "ign",
+		ingamename: "ign",
+		ign: function (target, room, user) {
+			if (!this.canTalk()) return false;
+			if (!target || target.length > 10) return this.errorReply(`Your IGN must be between 1-10 characters long.`);
+			let splatProfile = Db.splatoon.get(user.userid, {ranks: {}});
+			splatProfile.ign = target;
+			Db.splatoon.set(user.userid, splatProfile);
+			return this.sendReply(`Your In-Game Name has been set as: "${target}".`);
+		},
+
+		lvl: "level",
+		level: function (target, room, user) {
+			if (!this.canTalk()) return false;
+			if (!target || target > 99 || target < 1 || target.includes(".")) return this.errorReply(`Your level must be a number between 1-99 (no decimals).`);
+			let splatProfile = Db.splatoon.get(user.userid, {ranks: {}});
+			splatProfile.level = target;
+			Db.splatoon.set(user.userid, splatProfile);
+			return this.sendReply(`Your Level has been set to: Level ${target}.`);
+		},
+
+		profile: function (target, room, user) {
+			if (!this.runBroadcast()) return;
+			if (!target) target = user.userid;
+			target = toId(target);
+			if (target.length > 18) return this.errorReply("Usernames cannot exceed 18 characters.");
+			let self = this;
+			let targetUser = Users.get(target);
+			let username = (targetUser ? targetUser.name : target);
+			let splatProfile = Db.splatoon.get(toId(username), {ranks: {}});
+
+			function IGN() {
+				if (!splatProfile.ign) return ``;
+				return ` (<strong>IGN:</strong> ${splatProfile.ign})`;
+			}
+
+			function splatLevel() {
+				if (!splatProfile.level) return ``;
+				return ` <strong>Level:</strong> ${splatProfile.level}`;
+			}
+
+			let profile = ``;
+			profile += `<div><strong>Name:</strong> ${WL.nameColor(toId(username), true, true)}${IGN(toId(username))}${splatLevel(toId(username))}<br />`;
```

Comment:
Consolidate this template string into the original declaration of `profile`.
Basically, just do:
```js
let profile = `<div><strong>[rest of L201...]`;
```

Reply:
I was debating whether or not to do that